libs/sdk-ui-filters/src/RankingFilter/RankingFilterDropdownBody.tsx

```diff
@@ -37,6 +37,7 @@ interface IRankingFilterDropdownBodyComponentOwnProps {
     onDropDownItemMouseOver?: (ref: ObjRefInScope) => void;
```

Comment:
Please update the copyright of this file as well. As we are making some changes now, the copyright should change accordingly, shouldn't it?

`// (C) 2020 GoodData Corporation`

↓

`// (C) 2020-2022 GoodData Corporation`

Reply:
fixed